ICAFIX/hcp_fix_multi_run

```diff
@@ -838,91 +871,137 @@ fi
+		# User has specified a training data file
+		
+		# if training data is in legacy RData format, and not a built-in model, convert it
+    builtInModels="@HCP25_hp2000@HCP7T_hp2000@HCP_Style_Single_Multirun_Dedrift@NHP_HCP_Macaque@NHP_HCP_MacaqueCyno@Standard@UKBiobank@WhII_MB6@WhII_Standard@"
+		if [[ "${TrainingData}" == *.RData ]]; then 
+			TrainingData=${TrainingData%.RData}
+			if [[ ! ${builtInModels} == *"@${TrainingData}@"* ]]; then 
+				${FSL_FIXDIR}/convert_fix_model ${TrainingData}.RData ${TrainingData}.pyfix_model
+				TrainingData=${TrainingData}.pyfix_model
+			fi
+		fi
+	else
+		# User has not specified a training data file
+		log_Msg "training data file not specified, setting to HCP_Style_Single_Multirun_Dedrift"
+		TrainingData=HCP_Style_Single_Multirun_Dedrift
 	fi
-fi
 
-log_Msg "using training data file: ${TrainingData}"
+  # Alert user to log file location
+	log_Msg "pyfix log at ${ConcatFolder}/pyfix.log"
+fi #if ((legacyFix));then
 
 # set up fix command
 # use array for whitespace safety, even if the rest of the script isn't
 if ((doMotionRegression)); then
-	fix_cmd=("${FSL_FIXDIR}/fix" "${concatfmrihp}.ica" "${TrainingData}" "${FixThresh}" -m -h "${AlreadyHP}")
+  fix_cmd=("${FSL_FIXDIR}/fix" "${concatfmrihp}.ica" "${TrainingData}" "${FixThresh}" -m -h "${AlreadyHP}")
 else
-	#-h is actually a subargument to -m, and will cause problems if specified without (or even not directly following) -m
-	# Update (11/8/2019): No longer true as of FIX 1.06.12, but since filtering of the CIFTI in MR-FIX occurs in
-	# functionhighpassandvariancenormalize it doesn't matter, so we leave the code unchanged so that we don't
-	# need to add a FIX version dependency to the script
-	fix_cmd=("${FSL_FIXDIR}/fix" "${concatfmrihp}.ica" "${TrainingData}" "${FixThresh}")
+  #-h is actually a subargument to -m, and will cause problems if specified without (or even not directly following) -m
+  # Update (11/8/2019): No longer true as of FIX 1.06.12, but since filtering of the CIFTI in MR-FIX occurs in
+  # functionhighpassandvariancenormalize it doesn't matter, so we leave the code unchanged so that we don't
+  # need to add a FIX version dependency to the script
+  fix_cmd=("${FSL_FIXDIR}/fix" "${concatfmrihp}.ica" "${TrainingData}" "${FixThresh}")
 fi
+
+log_Msg "using training data file: ${TrainingData}"
 log_Msg "fix_cmd: ${fix_cmd[*]}"
-## MPH: The 'fix' script itself will continue to log to its own custom files
-## Alert user to where those are
-log_Msg "Check ${concatfmrihp}.ica/fix/logMatlab.txt for log output from feature extraction"
-log_Msg "Check ${concatfmrihp}.ica/.fix_2b_predict.log for log output from component classification"
-log_Msg "Check ${concatfmrihp}.ica/.fix.log for log output from cleanup stage"
 "${fix_cmd[@]}"
 return_code=$?
 if [ "${return_code}" -ne "0" ]; then
-	log_Err_Abort "return_code from fix_cmd: ${return_code}"
+  log_Err_Abort "return_code from fix_cmd: ${return_code}"
```

Comment:
Not sure why you changed the indentation here (considering the rest of the file is mostly tabs), as there doesn't seem to be logic changes other than reporting the training data and no longer reporting those log locations (are all of them defunct in pyfix?).  `git blame` and similar will see these indentation edits instead of the ones that actually changed the logic...